modules/encoders/php/base64.rb

```diff
@@ -20,12 +20,21 @@ def initialize
   end
 
   def encode_block(state, buf)
-    # Have to have these for the decoder stub, so if they're not available,
-    # there's nothing we can do here.
     %w[c h r ( ) . e v a l b a s e 6 4 _ d e c o d e ;].uniq.each do |c|
       raise BadcharError if state.badchars.include?(c)
     end
 
+    executor = "eval"
+    %w[e v a l].uniq.each do |c|
```

Comment:
I'm not entirely sure this is needed. In particular, the earlier check on line 25/23 will already check that `eval` isn't a bad entry, otherwise by this point you would have raised a `BadcharError` here.